telecine/src/main/java/com/jakewharton/telecine/TelecineTileService.java

```diff
@@ -49,4 +53,16 @@
         .setAction(Analytics.ACTION_QUICK_TILE_REMOVED)
         .build());
   }
+
+  @Subcomponent(modules = TelecineTileService_Module_ContributeTelecineTileService.class)
+  interface Component {
+    void inject(TelecineApplication app);
+
+    DispatchingAndroidInjector<Service> injector();
+  }
+
+  @dagger.Module
+  abstract class Module {
```

Comment:
this is the saddest part i see. this module is unused, and i don't know if there's anything to do about it.

Reply:
actually, let's not reference the generated module?

Reply:
done.

Reply:
(i used to be doing `@Subcomponent(modules = TelecineTileService_Module_ContributeTelecineTileService.class)`.)